charts/codesealer/README.md

```diff
@@ -13,33 +13,44 @@ In the following we assume that the access token is set in the following way:
 export CODESEALER_TOKEN=<access token>
 ```
 
+This installation requires a Kubernetes Cluster with kubectl.  
+
+### Ingress
+
+To use this Helm chart you will also need to set the following variables to match
+your Ingress Controller's deployment on your Kubernetes Cluster:
+
+```bash
+export INGRESS_NAMESPACE=ingress-nginx
+export INGRESS_DEPLOYMENT=ingress-nginx-controller
+export INGRESS_PORT=443
+export INGRESS_HELM_REPO=https://kubernetes.github.io/ingress-nginx
+export INGRESS_HELM_CHART=ingress-nginx
+```
+
 Additionally you will need an ingress and an application to protect. Below are steps to
 get started with a demo application and an Nginx Ingress. For guides on how to use this
 Helm chart with specific Kubernetes implementations, see the ["Kubernetes Implementation
 Specifics"](#kubernetes-implementation-specifics) section.
 
-### Ingress
-
 This Helm chart will install Codesealer as a sidecar to an existing ingress deployment.
-If you don't have an ingress already, you can install an [Nginx Ingress
-Controller](https://docs.nginx.com/nginx-ingress-controller/) using the following
-command:
+
+If you don't have an ingress already, you can install an [Nginx Ingress Controller](https://github.com/kubernetes/ingress-nginx/tree/main/charts/ingress-nginx) 
+using the following command:
 
 ```bash
-helm upgrade --install ingress-nginx ingress-nginx \
-  --repo https://kubernetes.github.io/ingress-nginx \
-  --namespace ingress-nginx --create-namespace \
-  --wait --timeout=60s
+helm repo add ${INGRESS_HELM_CHART} ${INGRESS_HELM_REPO}
```

Comment:
Why was this changed from upgrade to install?

Reply:
Originally we were bypassing the repo install.  This now matches what I have in the test script.